src/main/java/com/olgaboiar/mint/Header.java

```diff
@@ -48,7 +48,7 @@ String prepareHeaders() {
         return headersToSend;
     }
 
-    public void setAllowMethods(Set<String> allowedMethods1) {
+    public void setAllowMethods(String[] allowedMethods1) {
         allowedMethods = allowedMethods1;
```

Comment:
Here you can do:

```Java
this.allowedMethods = allowedMethods;
```

Reply:
https://github.com/olgaboiar/MintHTTPServer/pull/9/commits/82b52240f7bd8f93595ae4079bb1c8b434710657